lightning/src/ln/peer_handler.rs

```diff
@@ -115,6 +115,8 @@ struct Peer {
 	pending_read_is_header: bool,
 
 	sync_status: InitSyncTracker,
+
+ 	awaiting_pong: bool,
```

Comment:
nit: You have a whitespace beginning of the line there ?

You can add `exec git diff-index --check --cached $against --` in the default `.git/hooks/pre-commit` to avoid this in the future.

Reply:
sweet thanks, I will make a note of that.

I will go ahead and update it then rebase and push back

Reply:
I made some changes to the default in the text editor (I remembered about git diff --check) also

I am not sure about the whitespace though? do you mean just the additional space on 118?